portal/src/main/webapp/WEB-INF/jsp/global/js_include.jsp

```diff
@@ -65,6 +65,12 @@
 <script type="text/javascript" src="js/lib/plotly.min.js?<%=GlobalProperties.getAppVersion()%>"></script>
 <script type="text/javascript" src="js/lib/igv_webstart.js?<%=GlobalProperties.getAppVersion()%>"></script>
 <script type="text/javascript" src="js/lib/genomespace.js"></script>
+<script type="text/javascript" src="js/lib/handlebars.min.js"></script>
+<script type="text/javascript" src="js/lib/rangy-core.js"></script>
+<script type="text/javascript" src="js/lib/rangy-selectionsaverestore.js"></script>
+<script type="text/javascript" src="js/lib/rangy-textrange.js"></script>
+<script type="text/javascript" src="js/lib/jquery.caret.js?<%=GlobalProperties.getAppVersion()%>"></script>
```

Comment:
are all of these libraries used and not replaceable by home-rolled code? It's a lot to add to the global include, if we can avoid it.